### DISCUSSION TITLE
What is Non Indexed Locations

### DISCUSSION THREAD

**Discussion by Raghav-45** [spacedriveapp/spacedrive#1176] [Community Support] [answered]
Dear developers, I am eager to contribute to the project, and to better understand it, could you kindly explain the concept of `indexed` & `non-indexed` locations within the context of the project?

Having a clear understanding of this aspect will undoubtedly boost my confidence and enable me to make meaningful contributions.

**Comment by HeavenVolkoff** [comment, marked as answer]
Non-indexed, or ephemeral, locations represent directories on your local machine that you can view and navigate through in Spacedrive without the need to register them in the application beforehand. These locations allow you to start using Spacedrive immediately after installation, providing similar features as any other conventional file explorer. However, to leverage our VDFS (virtual distributed filesystem), you would need to create an indexed location from an ephemeral directory. This step enables offline and remote traversal of your files, strong per file/directory encryption, file sharing between your connected nodes, quick search results, and all the additional functionalities that Spacedrive aims to offer beyond the regular file explorer experience. We designed it this way to provide a seamless transition into the full range of Spacedrive's capabilities for a new user. I hope this fully answers your question.
